governance/doc.go

```diff
@@ -14,5 +14,53 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
 
-// Package governance contains functions and variables used for voting and reflecting vote results in Klaytn.
+/*
+Package governance contains functions and variables used for voting and reflecting vote results in Klaytn.
+In Klaytn, various settings such as the amount of KLAY minted as a block reward can be changed by using governance vote.
+These votes can be casted by nodes of Governance Council and detailed introduction can be found at <https://docs.klaytn.com/klaytn/governance>
+
+How to cast a vote
+
+To cast a vote, a node have to be a member of the Governance Council.
+If the governance mode is "single", only one designated node (the governing node) can vote.
+In the console of the node, "governance.vote(key, value)" API can be used to cast a vote.
+
+Keys for the voting API
+
+Following keys can be handled as of 7/20/2019.
+  - "governance.governancemode"   : To change the governance mode
+  - "governance.governingnode"    : To change the governing node if the governance mode is "single"
+  - "governance.unitprice"        : To change the unitprice of Klaytn (Unit price is same as gasprice in Ethereum)
+  - "governance.addvalidator"     : To add new node as a council node
+  - "governance.removevalidator"  : To remove a node from the governance council
+  - "istanbul.epoch"              : To change Epoch, the period to gather votes
+  - "istanbul.committeesize"      : To change the size of the committee
+  - "reward.mintingamount"        : To change the amount of block generation reward
+  - "reward.ratio"                : To change the ratio used to distribute the reward between block proposer node, PoC and KIR
+  - "reward.useginicoeff"         : To change the application of gini coefficient to reduce gap between CCOs
+  - "reward.deferredtxfee"        : To change the way of distributing tx fee
+  - "reward.minimumstake"         : To change the minimum amount of stake to participate in the governance council
+
+
+How governance works
+
+Governance package contains a governance struct which stores current system configurations and voting status.
+If a vote passed, the governance struct is updated to provide new information to related packages and users.
+The API documentation can be found at <https://docs.klaytn.com/api/governance>
+
+When a CN (consensus node which is managed by CCO) propose a block, it write its vote on the block header and other nodes
```

Comment:
propose -> proposes

Reply:
write -> writes

Reply:
Thanks for pointing it out! I made a new PR to fix it. Please take a look #70